### DISCUSSION TITLE
DgsDataFetching Environment at HttpServletRequest

### DISCUSSION THREAD

**Discussion by shikhar-multiplier** [Netflix/dgs-framework#1876] [Q&A]
Hi Everyone,

I am trying to implement Attribute-based Access Control that involves some business logic to be done before request lands on mutations and queries. I need to access the DgsDataFetchingEnvironment Object at the filter level. Can someone guide me if this can be done and how?

**Comment by paulbakker** [comment]
I don't think you can, a filter would happen before anything GraphQL happens, so there is no context yet.
